packages/sui-test/bin/karma/babelPatch.js

```diff
@@ -25,12 +22,17 @@ module.exports = function({types: t}) {
           const contextToUse = callee.property.name
           // extract category and component from the filename
           // by reversing the path and exlcuding the first slash (it's /index.js)
-          const [, component, category] = state.file.opts.filename
-            .split('/')
-            .reverse()
-          // we create a key based on the category and component path
-          const componentKey = `${category}/${component}`
-
+          const reversedPath = state.file.opts.filename.split('/').reverse()
+          const [, lastPath] = reversedPath
+          const isNewPath = lastPath === 'test'
+          let componentKey = ''
+          if (isNewPath) {
+            const [, , component, category] = reversedPath
```

Comment:
Maybe a better naming like isRelativeTestDirectoryPath more declarative. When do 'newPath' name becomes oldPath? 🥸

Reply:
A newer path available will become newNewPath or maybe theNewestPath

Reply:
It's fine, but the idea is to kill one of the paths (the old one).